aztec/src/main/kotlin/org/wordpress/aztec/AztecTagHandler.kt

```diff
@@ -34,26 +34,41 @@ class AztecTagHandler : Html.TagHandler {
     private class Li
     private class Strike
 
-    override fun handleTag(opening: Boolean, tag: String, output: Editable, xmlReader: XMLReader) {
+    override fun handleTag(opening: Boolean, tag: String, output: Editable, xmlReader: XMLReader) : Boolean {
         if (opening) {
             if (tag.equals(BULLET_LI, ignoreCase = true)) {
                 if (output.length > 0 && output[output.length - 1] != '\n') {
                     output.append("\n")
                 }
                 start(output, Li())
-            } else if (tag.equals(STRIKETHROUGH_S, ignoreCase = true) || tag.equals(STRIKETHROUGH_STRIKE, ignoreCase = true) || tag.equals(STRIKETHROUGH_DEL, ignoreCase = true)) {
+                return true
+            } else if (tag.equals(STRIKETHROUGH_S, ignoreCase = true)
+                    || tag.equals(STRIKETHROUGH_STRIKE, ignoreCase = true)
+                    || tag.equals(STRIKETHROUGH_DEL, ignoreCase = true)) {
                 start(output, Strike())
+                return true
+            } else if (tag.equals(BULLET_UL, ignoreCase = true)) {
+                // no op
+                return true
             }
         } else {
             if (tag.equals(BULLET_LI, ignoreCase = true)) {
                 if (output.length > 0 && output[output.length - 1] != '\n') {
                     output.append("\n")
                 }
                 end(output, Li::class.java, BulletSpan())
-            } else if (tag.equals(STRIKETHROUGH_S, ignoreCase = true) || tag.equals(STRIKETHROUGH_STRIKE, ignoreCase = true) || tag.equals(STRIKETHROUGH_DEL, ignoreCase = true)) {
+                return true
+            } else if (tag.equals(STRIKETHROUGH_S, ignoreCase = true)
+                    || tag.equals(STRIKETHROUGH_STRIKE, ignoreCase = true)
+                    || tag.equals(STRIKETHROUGH_DEL, ignoreCase = true)) {
                 end(output, Strike::class.java, StrikethroughSpan())
+                return true
+            } else if (tag.equals(BULLET_UL, ignoreCase = true)) {
```

Comment:
Yep, that would produce the same effect, I wasn't sure. I opted for keeping a separated list of "known but ignored tags" and a second "supported tags that does nothing".

I think we could add HTML formatting there.